src/App.scss

```
@@ -1,40 +1,11 @@
 @import '~bootstrap/scss/bootstrap';
 
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
+a.social {
```

```suggestion
.footer-link {
```

Good practice in CSS is to prefer using classes over html elements. Also, this selector is used by the Developer links as well as the Social links in the footer so the naming is not accurate. We can change the selector to something more descriptive like `.footer-link`